plugins/inject-base.js

```diff
@@ -0,0 +1,7 @@
+const origin = process.env.SITE_HOST ||
+    (typeof (window) !== 'undefined' ? window.location.origin : '/');
+const base = origin + (origin.endsWith('/') ? '' : '/');
+
+export default function (ctx, inject) {
+    inject('base', base);
```

Comment:
```suggestion
    inject('baseUrl', base);
```
Let's make this a bit less ambiguous (haven't suggested all the replacements needed)

Reply:
Changed the name in c2f7b01